### DISCUSSION TITLE
Xcode 14 drops deployment targets macOS 10.12 (Sierra) and earlier

### DISCUSSION THREAD

**Discussion by bryanc-nz** [sparkle-project/Sparkle#2194] [Q&A] [answered]
With Xcode 14 it is no longer possible to build software for macOS 10.12 or earlier. I have users who still run my software on these dropped versions.

Is there any attribute that can be specified in the Sparkle appcast xml to prevent users of the earlier systems from being notified when I release a new version of my software with a minimum deployment target of macOS 10.13?

**Comment by zorgiepoo** [comment, marked as answer]
Check [minimum system version requirements](https://sparkle-project.org/documentation/publishing/#minimum-system-version-requirements) in the documentation. If you use generate_appcast it should automatically populate `sparkle:minimumSystemVersion` based on your app's specified `LSMinimumSystemVersion` in the Info.plist.

**Reply by bryanc-nz** [reply]
Great. Thank you very much for the prompt response.
